hack/netpol/Dockerfile

```diff
@@ -4,6 +4,6 @@ ADD ./ /netpol
 WORKDIR /netpol
 RUN  go build -o main pkg/main/main.go
 
-FROM alpine
+FROM scratch
```

Comment:
I suggested doing this in a separate PR

Reply:
I see, will revert that change, thx!

Reply:
Done, will put the change for `netpol` image in another PR.